lib/game.rb

```diff
@@ -0,0 +1,80 @@
+require 'pry'
+class Game
+#  include Players::Human
+  attr_accessor :board, :player_1, :player_2, :token
+
+  WIN_COMBINATIONS = [
+    [0,1,2],  #Top row
+    [3,4,5],  #Middle row
+    [6,7,8],  #Bottom row
+    [0,3,6],  #First column
+    [1,4,7],  #Second column
+    [2,5,8],  #Third column
+    [0,4,8],  #First diagonal
+    [2,4,6],  #Second diagonal
+  ]
+## How does that work ?? - arguments? without including modules?
+  def initialize(player_1 = Players::Human.new("X") , player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+
+  def current_player
+    if @board.turn_count % 2 == 0
+      @player_1
+    else
+      @player_2
+    end
+  end
+
+  def won?
+    WIN_COMBINATIONS.detect do |winning_combination|
+      position_1 = @board.cells[winning_combination[0]]
+      position_2 = @board.cells[winning_combination[1]]
+      position_3 = @board.cells[winning_combination[2]]
+      position_1 == position_2 && position_2 == position_3 && position_1 != " "
+    end
+  end
+
+  def draw?
+    won? == nil && @board.full? == true
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def winner
+    if won?
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.